components/collection/drop/GenerativeLayout.vue

```diff
@@ -0,0 +1,102 @@
+            v-if="!isMobile"
+            :has-user-minted="hasUserMinted"
+            :is-wallet-connecting="isWalletConnecting"
+            :is-image-fetching="isImageFetching"
+            :is-loading="isLoading"
+            :minimum-funds="minimumFunds"
+            :minimum-funds-description="minimumFundsDescription"
+            :max-count="maxCount"
+            :minted-count="mintedCount"
+            :mint-count-available="mintCountAvailable"
+            :mint-button-disabled="mintButtonDisabled"
+            :mint-button-label="mintButtonLabel"
+            :is-holder-of-target-collection="isHolderOfTargetCollection"
+            :holder-of-collection-id="holderOfCollectionId"
+            @mint="handleSubmitMint" />
+        </div>
+
+        <div class="column pt-5 is-flex is-justify-content-center">
+          <CollectionDropGenerativePreview
+            :content="drop.content"
+            :image="drop.image"
+            @select="handleSelectImage" />
+        </div>
+
+        <CollectionDropMintSection
+          v-if="isMobile"
+          class="column"
+          :has-user-minted="hasUserMinted"
+          :is-wallet-connecting="isWalletConnecting"
+          :is-image-fetching="isImageFetching"
+          :is-loading="isLoading"
+          :minimum-funds="minimumFunds"
+          :minimum-funds-description="minimumFundsDescription"
+          :max-count="maxCount"
+          :minted-count="mintedCount"
+          :mint-count-available="mintCountAvailable"
+          :mint-button-disabled="mintButtonDisabled"
+          :mint-button-label="mintButtonLabel"
+          :is-holder-of-target-collection="isHolderOfTargetCollection"
+          :holder-of-collection-id="holderOfCollectionId"
+          @mint="handleSubmitMint" />
+      </div>
+
+      <CollectionUnlockableItemInfo :collection-id="collectionId" />
+      <div class="my-4">
+        <CarouselTypeLatestMints
+          :collection-id="collectionId"
+          interaction="MINT" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import UnlockableCollectionInfo from '@/components/collection/unlockable/UnlockableCollectionInfo.vue'
+
+withDefaults(
+  defineProps<{
+    collectionId: string
+    description?: string
+    drop: any
+    mintButtonDisabled: boolean
+
+    mintedCount: number
+    mintCountAvailable: boolean
+    maxCount: number
+    minimumFunds: number
+    minimumFundsDescription: string
+    isImageFetching: boolean
+    isWalletConnecting: boolean
+    isLoading: boolean
+    mintButtonLabel: string
+    hasUserMinted?: string
+    isHolderOfTargetCollection?: boolean
+    holderOfCollectionId?: string
+
+    handleSelectImage: (image: string) => void
+    handleSubmitMint: () => void
+  }>(),
+  {
+    hasUserMinted: undefined,
+    isHolderOfTargetCollection: false,
+    holderOfCollectionId: '',
+  },
+)
+
+const { width } = useWindowSize()
+const isMobile = computed(() => width.value <= 768)
```

Comment:
maybe use css helpers to toogle visiblity?

Reply:
will do  https://github.com/kodadot/nft-gallery/issues/8767 here if available